app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return f"{{Name: {self.name}," \
+               f" Health: {self.health}," \
+               f" Hidden: {self.hidden}}}"
+
+    def __repr__(self) -> str:
+        return str(self)
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
+        if self.hidden is True:
+            self.hidden = False
+        elif self.hidden is False:
+            self.hidden = True
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(victim: Herbivore) -> None:
+        if isinstance(victim, Herbivore)\
+                and victim.health > 0 and not victim.hidden:
```

Comment:
There is no need to check that `victim` has health greater than 0, you could omit it because you redefine health on line 40. Besides that, do not use line breaking `\` in the if condition.